packages/valory/skills/registration_abci/behaviours.py

```diff
@@ -332,11 +289,33 @@ def request_tendermint_info(self) -> Generator[None, None, bool]:
             self.collection_complete = True
         return self.collection_complete
 
+    def format_genesis_data(
+        self,
+        collected_agent_info: Dict[str, Any],
+    ) -> Dict[str, Any]:
+        """Format collected agent info for genesis update"""
+
+        validators = []
+        for i, validator_config in enumerate(collected_agent_info.values()):
+            validator = dict(
+                address=validator_config["address"],
+                pub_key=validator_config["pub_key"],
+                power=self.params.voting_power,
+                name=f"node{i}",
+            )
+            validators.append(validator)
```

Comment:
We should use a mapping from agent to node name instead of relying on being able to enumerate what is assumed here to be ordered. This may not be an issue for communication between the nodes, but I can image that running containers named node0-3 might no longer correspond which may lead to confusion on the developers side as well.